### ISSUE TITLE
Instructions to update mongoose

### ISSUE THREAD

**Issue by speterli** [gavinaiken/mongoose-browser#2]
Are there any instructions how to update to newest mongoose version?

**Comment by gavinaiken** [comment]
No, I would need to update the repo. Are you aware of any new features in the latest which are not in the currently published one?

**Comment by gavinaiken** [comment]
I'm currently using this version in the browser in conjunction with 4.9.3 on the server side with no issues. The browser functionality is quite a small subset of the server side capabilities.

**Comment by speterli** [comment]
Thanks for reporting. 
Actually there is an issue but currently not fixed.

https://github.com/Automattic/mongoose/issues/5440

**Comment by gavinaiken** [comment]
OK, I can update this repo as soon as that issue is fixed and released.
